spec/migrate/001_tables.rb

```diff
@@ -1,5 +1,11 @@
 Sequel.migration do
   up do
+    primary_key_type = if ENV['RODAUTH_SPEC_UUID'] && database_type == :postgres
```

Comment:
Style-wise, I would prefer the ternary operator on a single line in this case (and the other spec migration).

Reply:
I've updated the three files with this check and converted them to ternary operators on a single line.